code/modules/projectiles/projectile/bullets.dm

```diff
@@ -164,6 +164,28 @@
 	..()
 	SpinAnimation()
 
+// new dildo bullet! adds horniness - basically just the magic banana without the stun, and inflicts arousalloss
+/obj/item/projectile/bullet/dildo
+	damage = 0
+	forcedodge = 1
+	nodamage = 1
+	name = "dildo"
+	icon = 'code/citadel/icons/dildo.dmi'
+	icon_state = "dildo_knotted_3"
+	range = 200
+
+/obj/item/projectile/bullet/dildo/on_hit(atom/target, blocked = 0)
+	. = ..()
+	if(iscarbon(target))
+		var/mob/living/carbon/M = target
+		M.arousalloss += 5
+		M.update_arousal_hud()
```

Comment:
M.adjustArousalLoss(5)

Might also wanna add a check around here to see if arousal's even enabled, so people don't get the "You feel slightly hornier..." message if they've got arousal disabled.